cpp/lib/src/MARC.cc

```diff
@@ -2937,10 +2937,42 @@ std::set<std::string> ExtractPrintCrossLinkPPNs(const MARC::Record &record) {
 }
 
 
-std::set<std::string> ExtractPrintAndOnlineCrossLinkPPNs(const MARC::Record &record) {
+std::set<std::string> ExtractOtherCrossLinkPPNs(const MARC::Record &record) {
+    std::set<std::string> cross_reference_ppns;
+    for (const auto _785_field : record.getTagRange("785")) {
+        const auto ppn(BSZUtil::GetK10PlusPPNFromSubfield(_785_field, 'w'));
+        if (ppn.empty())
+            continue;
+
+        const auto subfield_i_contents(_785_field.getFirstSubfieldWithCode('i'));
+        if (subfield_i_contents == "Forts." or subfield_i_contents == "Fortgesetzt durch") {
+            cross_reference_ppns.emplace(ppn);
+            continue;
+        }
+    }
+
+    for (const auto _780_field : record.getTagRange("780")) {
+        const auto ppn(BSZUtil::GetK10PlusPPNFromSubfield(_780_field, 'w'));
+        if (ppn.empty())
+            continue;
+
+        const auto subfield_i_contents(_780_field.getFirstSubfieldWithCode('i'));
+        if (subfield_i_contents == "Vorg.") {
+            cross_reference_ppns.emplace(ppn);
+            continue;
+        }
+    }
+
+    return cross_reference_ppns;
+}
+
+
+std::set<std::string> ExtractCrossLinkPPNs(const MARC::Record &record) {
     auto cross_reference_ppns(ExtractOnlineCrossLinkPPNs(record));
     const auto print_cross_reference_ppns(ExtractPrintCrossLinkPPNs(record));
     cross_reference_ppns.insert(print_cross_reference_ppns.cbegin(), print_cross_reference_ppns.cend());
+    const auto other_cross_reference_ppns(ExtractOtherCrossLinkPPNs(record));
```

Comment:
if the function is named ExtractCrossLinkPPNs, then why are the variables called cross_reference_ppns and not cross_link_ppns?